index.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+entries:
+  ark-activemq:
+  - apiVersion: v2
+    appVersion: 5.16.2
+    created: "2021-07-29T14:22:58.463485118+01:00"
+    description: Helm chart for ActiveMQ as used by ArkCase
+    digest: 1ae47ba33af768adf50afc4746c87b091cb98d4a35dbc55febaafb9bad12165d
+    keywords:
+    - activemq
+    - message broker
+    name: ark-activemq
+    sources:
+    - https://github.com/ArkCase/ark_helm_charts
+    urls:
+    - https://arkcase.github.io/ark_helm_charts/ark-activemq-0.1.0.tgz
+    version: 0.1.0
```

Comment:
Is this file a helm of helms?

Reply:
@JonHolman No, it's the index of the available Helm charts in this repository (in the sense of a Helm repository, not a GitHub repository). In essence, a Helm repository is just a website with a `index.yaml` file. 

When I add this Helm repository on my laptop using the `helm repo add` command, Helm will download this `index.yaml` file to know which Helm charts are available in this Helm repository. The `urls` parameter for each Helm chart shows the URL to download the Helm chart, which must be a .tgz package. That's why I have to commit the `ark-activemq-0.1.0.tgz` file in GitHub.

Once this PR is merged, I will configure GitHub pages for this repo so it will act as a web server and the `index.yaml` should be available at `https://arkcase.github.io/ark_helm_charts/index.yaml`. At least that's what I hope! (I am following a tutorial on the internet)

Reply:
Thanks @fabricetriboix